pytorch_lightning/trainer/properties.py

```diff
@@ -488,6 +489,14 @@ def train_loop(self) -> FitLoop:
         # FIXME(@awaelchli): the current train_loop should be renamed to fit_loop
         return self.fit_loop
 
+    @property
+    def evaluation_loop(self) -> EvaluationDataLoaderLoop:
+        if self.training and self.sanity_checking:
```

Comment:
This won't ever return True. You meant

`self.trainer.state.fn == TrainerFn.FIT and self.sanity_checking`

but no need for the first condition as sanity_checking is only done for fit

```suggestion
        if self.sanity_checking:
```

Reply:
Thanks ! I wasn't sure.